colorama/ansitowin32.py

```diff
@@ -21,6 +21,35 @@ def is_a_tty(stream):
     return hasattr(stream, 'isatty') and stream.isatty()
 
 
+def check_windows_ansi_support():
+    try:
+        import winreg
+    except ImportError:
+        try:
+            import _winreg  # on Python 2
+        except ImportError:
+            return False
+        else:
+            winreg = _winreg
+
+    # Check the registry for release ID
+    key = r"SOFTWARE\Microsoft\Windows NT\CurrentVersion"
+    val = r"ReleaseID"
+    key = winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE, key)
```

Comment:
Use `try finally` (It is a context manager since 2.6 but I think colorama still declares compatibility with Python 2.5).